src/utils/getFormattedLinks.ts

```diff
@@ -0,0 +1,21 @@
+import { FormatLink } from "../constants/types";
+
+function formatLinkData(link: FormatLink) {
+  return {
+    id: link.id,
+    imageSource: link.image_source,
+    createdAt: link.created_at,
+    description: link.description,
+    title: link.title,
+    url: link.url,
+  };
+}
+
+export default function getFormattedLinks(links: FormatLink[]) {
+  return (
+    links &&
+    links.map((link: FormatLink) => {
```

Comment:
타입이 자동으로 추론되는 곳에서는 잡아주지 않아도 괜찮습니다~
```suggestion
    links.map((link) => {
```